### PR TITLE
[SIMPLE_FORMS] fix update submission archive builder logic to fix bugs

### PR DESCRIPTION
**Pull request by pennja**
## Summary

- This work updates the submission archive builder in accordance with VBA direction
- This work also fixes minor bugs with may cause the builder to fail

## Related issue(s)

- [Create Solution for adding Form Submission Remediation Documents to an S3 bucket](https://app.zenhub.com/workspaces/vagov-product-team-forms-634853151f5c6000165942bc/issues/gh/department-of-veterans-affairs/va.gov-team-forms/1633)

## Testing done

- New code is NOT covered by unit tests YET. This is due to flaky tests bringing down the staging environment yesterday. Tests to come!

## Requested Feedback

Any

### CODE REVIEW

File: modules/simple_forms_api/app/services/simple_forms_api/s3/submission_archive_builder.rb [hunk 1]
```diff
@@ -49,9 +46,7 @@ def valid_submission_data?(data)
 
       def process_submission_files
         write_pdf
-        write_as_json_archive if include_json_archive
-        write_as_text_archive if include_text_archive
-        write_attachments unless attachments && attachments.empty?
+        write_attachments if attachments&.any?
```
Comment (Thrillberg) [inline modules/simple_forms_api/app/services/simple_forms_api/s3/submission_archive_builder.rb:49]: 👍

File: modules/simple_forms_api/lib/tasks/archive_forms_by_uuid.rake [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
 #  Set the parameters as variables ahead of time:
-#    rails simple_forms_api:archive_forms_by_uuid[benefits_intake_uuids, parent_dir]
+#    bundle exec rails simple_forms_api:archive_forms_by_uuid[benefits_intake_uuids, parent_dir]
 #  Pass in UUIDs only if default parent_dir is appropriate:
-#    rails simple_forms_api:archive_forms_by_uuid[abc-123 def-456]
-# Pass in new directory to override default:
-#    rails simple_forms_api:archive_forms_by_uuid[abc-123 def-456,custom-directory]
+#    bundle exec rails simple_forms_api:archive_forms_by_uuid[abc-123 def-456]
+#  Pass in new directory to override default:
+#    bundle exec rails simple_forms_api:archive_forms_by_uuid[abc-123 def-456,custom-directory]
```
Comment (Thrillberg) [inline modules/simple_forms_api/lib/tasks/archive_forms_by_uuid.rake:9]: Nice!

**Review by Thrillberg — APPROVED** [review]
Looks great! Thanks for all the little fixes along the way, too!

**Review by ryan-mcneil — APPROVED** [review]

**Comment by ryan-mcneil** [comment]
@pennja if you're unable to merge because of those failing checks, let me know and I can do it. Thanks!

**Comment by pennja** [comment]
> @pennja if you're unable to merge because of those failing checks, let me know and I can do it. Thanks!

@ryan-mcneil Looks like I'm not able to merge in unfortunately

**Pull request merged**